_posts/2018-08-20-rnp-010-released.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title:  "RNP 0.10.0 has been released"
+date:   2018-08-20 20:37:38 +0700
+categories: release
+excerpt: >-
+    After a year since it stemmed off NetPGP, the OpenPGP library
+    shares little code in common with its ancestor.
+    Enjoy new features, better performance, and better
+    compatibility with other implementations.
+---
+
+:cpp: C++
+
+Meet RNP, an  RFC4880-compliant OpenPGP library written in {cpp}.
+
+RNP was born at Hong Kong-based Ribose and is maintained under its initiative.
+Originally stemmed from NetPGP, it has little in common with its ancestor now
+after a year of development—after a clean-up of legacy code,
+corrected compatibility issues with GnuPG and other implementations,
+improved performance and added cutting-edge features.
+
+Feel free to use RNP in your software!
```

Comment:
This line feels a little out of place. I'm thinking it could be reworded to include mention of the BSD licensing though maybe?